public/stylesheets/app.css

```diff
@@ -156,17 +156,18 @@ body.dark-mode main a {
 	color: var(--dark-mode-fg-color);
 }
 
-.label.is-required::after, .select.is-required::after {
+.label.is-required::after {
 	content: "*";
 	color: red;
-	display: incline;
+	display: inline;
 	font-size: 1.2rem;
-	/* margin-left: -0.8em; */
 }
-.control.is-required::before {
-	content: "*";
-	color: red;
-	display: incline;
-	font-size: 1.2rem;
-	margin-left: -0.8em;
+
+.control.is-required::after {
+    content: "*";
+    color: red;
+    display: inline;
```

Comment:
For the `/spider` required field, how about just put it after the `<input>` element? Something like:

```css
.label.is-required::after, .input.is-required::after {
// ...
}
```

```erb
<input class="input is-required" type="text" name="target" placeholder="example.com OR www.example.com OR https://example.com/" value="<%=hattr params[:target] %>">
```

That way it is consistent with how `.label.is-required` works.

Reply:
I can't seem if we can do it because if we do that, that is, instead of adding it to `control` if we add that to `input`, the astrisk might not appear. This is because the ::before and ::after pseudo-elements don't work as expected on self-closing elements like `<input>, <img>, <br>, <hr>` and others.

HTML `<input>` elements are void elements (also known as self-closing elements), meaning they don't have an end tag and don't contain content. The CSS pseudo-elements `::before` and `::after` are used to insert content before or after the content of an element. Since `<input>` elements don’t have any content (they can't have child elements or text), these pseudo-elements have no effect when used with `<input>`. That's the reason I refrained from adding it to `.input.is-required`on first hand.

Reply:
Ah you are correct.